src/deadline/maya_adaptor/MayaClient/render_handlers/vray_handler.py

```
@@ -116,6 +107,43 @@ def start_render(self, data: dict) -> None:
         if maya.cmds.getAttr("vraySettings.sys_message_level") < 3:
             maya.cmds.setAttr("vraySettings.sys_message_level", 3)
 
+        # Perform setup for region rendering if needed, otherwise just use the output size from the submission
+        region = [
+            data.get(field) 
+            for field in ("region_min_x", "region_min_y", "region_max_x", "region_max_y")
+        ]
+        if any(v is not None for v in region):
+            print(f"MayaClient: Region bounds {region} specified.", flush=True)
+
+            region_minX, region_minY, region_maxX, region_maxY = region
+            region_str = f"(minX={region_minX}, minY={region_minY}, maxX={region_maxX}, maxY={region_maxY})"
+            if any(v is None for v in region):
+                raise RuntimeError(
+                    f"MayaClient: Region bounds {region_str} must be fully defined or all empty, but were partially specified."
+                )
+
+            # Set the output filename
+            maya.mel.eval(f'''setAttr -type "string" "vraySettings.fileNamePrefix" "{data.get("output_file_prefix")}";''')
+
+            # Set to allow region in batch rendering
+            maya.cmds.setAttr("vraySettings.vfbRgnOffBatch", 0)
+
+            # Set to use VFB
+            maya.cmds.setAttr("vraySettings.vfbOn", 1)
+
+            # Set the image format to EXR - This is needed to make sure image is properly cropped
```

It should probably raise an error if the image format is different from EXR, instead of overriding the format.

---

Good call, added that and tested with a png output and exr output job.